Sources/Core/Formatter/MultilineFormatter.swift

```diff
@@ -6,44 +6,50 @@
 //
 
 class MultilineFormatter: PrettyFormatter {
-    private let option: Debug.Option
+    private let indentSize: Int
```

Comment:
Refactor: `MultilineFormatter` depends on `indentSize` instead of `Debug.Option`.